packages/swapper/src/swappers/RelaySwapper/constant.ts

```diff
@@ -54,6 +54,14 @@ export const chainIdToRelayChainId = {
   [bscChainId]: bsc.id,
 }
 
+export enum RelayStatusMessage {
+  WaitingForDeposit = 'Waiting for deposit...',
+  DepositDetected = 'Deposit detected, processing request...',
```

Comment:
```suggestion
  DepositDetected = 'Deposit detected, processing swap...',
```

Reply:
Done in [180cbd0](https://github.com/shapeshift/web/pull/9389/commits/180cbd0e95d02a4926da953828c0e1ff46c7f695)